library/core/class.pluginmanager.php

```diff
@@ -784,15 +784,16 @@ public function callEventHandlers($sender, $eventClassName, $eventName, $eventHa
      * @return mixed Returns whatever the event handler returns or **false** of there is not event handler.
      */
     public function callEventHandler($sender, $className, $eventName, $handlerType = 'handler') {
-        $eventKey = strtolower("{$className}_{$eventName}");
-        $handlerType = strtolower($handlerType);
+        $eventKey = "{$className}_{$eventName}";
```

Comment:
If you check where this variable goes - you will find the path:
```
> $this->eventManager->fire( ...
```
and inside the fire very 1st line is:
```
$handlers = $this->getHandlers($event);
```
then check ```getHandlers``` again the very 1st line
```
$name = strtolower($name);
```
so imho - we don't need to call strtolower twice, since we have it at the end of chain.